<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchWindow</class>
 <widget class="QMainWindow" name="LaunchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="consoleMainContainerSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QSplitter::handle
{
    background-color: rgba(255, 255, 255, 80);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="mainContainer" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="templateRecentsSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSplitter::handle
{
    background-color: rgba(255, 255, 255, 80);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="handleWidth">
           <number>10</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="recentsWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="recentsLayout">
            <item>
             <layout class="QHBoxLayout" name="appTitleLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="logoLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../resources/logos.qrc">:/logos/logo_small.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>8</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="geometrizeTitle">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string notr="true">Geometrize</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>15</number>
                </property>
                <property name="leftMargin">
                 <number>15</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>15</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QToolButton" name="openImageButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">padding: 5px;</string>
                  </property>
                  <property name="text">
                   <string extracomment="Text on a button that opens an image selection dialog when it is pressed">Open Image</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../resources/icons.qrc">
                    <normaloff>:/icons/folder_image.png</normaloff>:/icons/folder_image.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                  <property name="arrowType">
                   <enum>Qt::NoArrow</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="openLinkButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">padding: 5px;</string>
                  </property>
                  <property name="text">
                   <string extracomment="Text on a button that opens a dialog that allows the user to enter a website link, which is then downloaded">Open Link</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../resources/icons.qrc">
                    <normaloff>:/icons/globe_network.png</normaloff>:/icons/globe_network.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="runScriptButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">padding: 5px;</string>
                  </property>
                  <property name="text">
                   <string extracomment="Text on a button that opens a dialog that allows the user to select and run a script file">Run Script</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../resources/icons.qrc">
                    <normaloff>:/icons/script_go.png</normaloff>:/icons/script_go.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="openTaskQueueButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">padding: 5px;</string>
                  </property>
                  <property name="text">
                   <string>Open Task Queue</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../resources/icons.qrc">
                    <normaloff>:/icons/paginator.png</normaloff>:/icons/paginator.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="recentsListTitle">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string extracomment="Title of a section that contains a list of files and items the user recently opened">Recent</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="geometrize::dialog::RecentTasksList" name="recentsList">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="templatesWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="templatesTitle">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string extracomment="Title of a section that contains specially prepared template or sample images and photographs">Sample Images</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="geometrize::dialog::CompletionBox" name="templatesSearchEdit">
                  <property name="text">
                   <string notr="true" extracomment="The contents of the search box the user can type in to search for recent demo projects"/>
                  </property>
                  <property name="placeholderText">
                   <string extracomment="Placeholder text in a search box, telling the user to type in the box to find sample/test images">Search sample images...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="moreResourcesLink">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string notr="true">More Resources Link</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                    <property name="textInteractionFlags">
                     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="tutorialsLink">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string notr="true">Tutorials Link</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                    <property name="textInteractionFlags">
                     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollableWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>816</width>
                 <height>496</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="geometrize::dialog::TemplateGrid" name="templateGrid" native="true">
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="geometrize::dialog::ScriptConsole" name="consoleWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string extracomment="The title of the File menu that contains commands relating to the handling of files, such as open, save, print, etc">File</string>
    </property>
    <addaction name="actionClear_Recents"/>
    <addaction name="actionGlobal_Preferences"/>
    <addaction name="actionLoad_Global_Preferences"/>
    <addaction name="actionSave_Global_Preferences"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string extracomment="The title of the Help menu that contains links to content to help users, such as tutorial videos">Help</string>
    </property>
    <addaction name="actionTutorials"/>
    <addaction name="actionSupport"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string extracomment="The title of the View menu that contains commands relating to the handling of visual elements, such as the size and visibility of windows and dialogs within the app">View</string>
    </property>
    <addaction name="actionScript_Console"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionClear_Recents">
   <property name="text">
    <string extracomment="Text on a menu item that clears a list of recently loaded files when pressed">Clear Recent Items</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string extracomment="Text on a menu item that closes the window when pressed">Exit</string>
   </property>
  </action>
  <action name="actionTutorials">
   <property name="text">
    <string extracomment="Text on a menu item that opens the tutorials page when pressed">Tutorials</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string extracomment="Text on a menu item that opens the customer support/help page when pressed">Support</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string extracomment="Text on a menu item that opens the About page with information about the app when pressed">About</string>
   </property>
  </action>
  <action name="actionScript_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string extracomment="Text on a menu item that shows or hides the script console when pressed">Script Console</string>
   </property>
  </action>
  <action name="actionGlobal_Preferences">
   <property name="text">
    <string extracomment="Text on a menu item that opens the application-wide/global preferences page">Global Preferences</string>
   </property>
  </action>
  <action name="actionLoad_Global_Preferences">
   <property name="text">
    <string extracomment="Text on a menu item that opens a dialog that allows the user to load a file containing application-wide/global preferences page">Load Global Preferences</string>
   </property>
  </action>
  <action name="actionSave_Global_Preferences">
   <property name="text">
    <string extracomment="Text on a menu item that opens a dialog that allows the user to save a file containing the current application-wide/global preferences page">Save Global Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>geometrize::dialog::CompletionBox</class>
   <extends>QLineEdit</extends>
   <header>dialog/completionbox.h</header>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::ScriptConsole</class>
   <extends>QWidget</extends>
   <header>dialog/scriptconsole.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::RecentTasksList</class>
   <extends>QListWidget</extends>
   <header>dialog/recenttaskslist.h</header>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::TemplateGrid</class>
   <extends>QWidget</extends>
   <header>dialog/templategrid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
  <include location="../../resources/logos.qrc"/>
 </resources>
 <connections/>
</ui>
